src/main/java/tictactoe/Display.java

```diff
@@ -18,7 +22,7 @@ public void showGrid(Grid grid) {
 
     public void outcomeMessage(String outcome) {
         clearScreen();
-        if (outcome.equals("tie")) {
+        if (outcome.equals("TIE")) {
```

Comment:
I'd be interested to know what your reasons were for choosing not to use the enum here?

Reply:
Thanks for spotting this! I'll change it